CODEOWNERS

```diff
@@ -1,16 +1,10 @@
 # These datasets are subject to the additional change control procedures
 # described in https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/
 # Active Users
-/sql_generators/active_users_aggregates_v3/templates/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/firefox_desktop_derived/active_users_aggregates_v3 @mozilla/kpi_table_reviewers
-/sql_generators/active_users_aggregates_v4/templates/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/firefox_desktop_derived/active_users_aggregates_v4/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/fenix_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/firefox_ios_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/focus_android_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/focus_ios_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/klar_ios_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
-/sql/moz-fx-data-shared-prod/klar_android_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
+sql_generators/active_users_aggregates_v3/templates/ @mozilla/kpi_table_reviewers
```

Comment:
@BenWu I'm not sure about the starting `/`, is the intention here that we're expecting these directories to exist in the root directory?

Reply:
Yes without the `/` it will match any path in the repo even if it's nested